backup/start-metrics.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -ex
+
+cat <<EOF > config.json
+{
+  "labels": {
+    "db_name": "montagu"
+  },
+  "cache_volume": "barman_metrics",
+  "cache_filename": "metrics.json",
+  "port": 5000,
```

Comment:
Port should be configurable/passed in as an option, unless the idea is to only use this script on EC2, where we're sure port 5000 is available?